dev/integration_tests/ios_host_app/Host/AppDelegate.h

```diff
@@ -2,12 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import <UIKit/UIKit.h>
-#import <Flutter/Flutter.h>
+@import UIKit;
+@import Flutter;
+
+NS_ASSUME_NONNULL_BEGIN
 
 @interface AppDelegate : FlutterAppDelegate
 
-@property(nonatomic, strong) FlutterEngine* engine;
-@property(nonatomic, strong) FlutterBasicMessageChannel* reloadMessageChannel;
+@property(readonly, nullable) FlutterEngine* engine;
```

Comment:
I would retain `nonatomic` across the board.  Not a huge deal for integration tests it's nice to keep the semantics for the usage.  Also it means you have to change the header if you decide to have a custom setter or getter.